### ISSUE TITLE
Document Cloudflare and Fastly proxy setup

### ISSUE THREAD

**Issue by dbu** [FriendsOfSymfony/FOSHttpCache#546]
Add chapters to the doc at https://foshttpcache.readthedocs.io/en/latest/proxy-configuration.html to explain how to set up fastly resp cloudflare to work with FOSHttpCache.

lets not make general fastly and cloudflare tutorials, but explain only things specific for FOSHttpCache, like the fastly vcl for cache tags and such (that is probably quite similar to the varnish proxy configuration chapter)

**Comment by dbu** [comment]
@simonrjones as the original contributor, would you be motivated to provide that documentation for cloudflare?

@andrerom as the original contributor, would you be motivated to provide that documentation for fastly?

**Comment by andrerom** [comment]
Unfortunately I didn't work on Symfony nor Fastly the last 3 years, so I don't have any setup or fresh experience with either to get started on that.  I might be able to review, but limitations from above applies 😅

**Comment by simonrjones** [comment]
> @simonrjones as the original contributor, would you be motivated to provide that documentation for cloudflare?

Yes, I can do this. May be a few weeks until I get time to review.

**Comment by DemigodCode** [comment]
https://github.com/FriendsOfSymfony/FOSHttpCache/pull/547

I believe CloudFlare is working almost the same way as Fastly.
Did evaluate both the last 2 months

**Comment by dbu** [comment]
setting up the clients is documented already, but we miss documentation on how to configure the services for working with the concepts in FOSHttpCache.

fastly #547
cloudflare #516
